src/NosCore.GameObject/Character.cs

```diff
@@ -1427,7 +1430,7 @@ WearableInstance fairy
                 && (sp.Item.Element != 0) && (fairy.Item.Element != sp.Item.Element)
                 && (fairy.Item.Element != sp.Item.SecondaryElement))
             {
-                SendPacket(new MsgPacket
+                await SendPacket(new MsgPacket
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)